### DISCUSSION TITLE
Issues with images in Wiki-Page

### DISCUSSION THREAD

**Discussion by galdo** [jserodio/rg351p-ppsspp-settings#15] [General]
Hi - I got some issues, not loading images placed in the wiki page:

https://github.com/jserodio/rg351p-ppsspp-settings/wiki

![Bildschirmfoto 2021-05-16 um 10 43 50](https://user-images.githubusercontent.com/103244/118391253-a6070f80-b633-11eb-8d14-3066bff69ab8.png)

**Comment by jserodio** [comment]
Yeah it's an issue right now. Something broke but I'm not sure what.
It should display a green dot. I need to investigate this further, thanks for reporting it

**Comment by jserodio** [comment]
Nevermind I just found out that placehold.it is not renewing their SSL certificate, it may be related to moving to another hosting. It seems that placehold.it is now placeholder.com

This broken
https://placehold.it/15/008000/000000?text=+

This works
https://via.placeholder.com/15/008000/000000?text=+

I'll fix it ASAP, I'm in mobile right now

Regards

**Comment by jserodio** [comment]
Fixed, thanks @galdo for reporting it.
